tests/ert/unit_tests/gui/simulation/test_run_dialog.py

```diff
@@ -708,3 +710,53 @@ def test_that_stdout_and_stderr_buttons_react_to_file_content(
 
         with qtbot.waitSignal(run_dialog.accepted, timeout=30000):
             run_dialog.close()
+
+
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "design_matrix_entry",
+    (True, False),
+)
+def test_that_design_matrix_show_parameters_button(
+    design_matrix_entry, qtbot: QtBot, storage
+):
+    xls_filename = "design_matrix.xls"
+    with open(f"{xls_filename}", "w", encoding="utf-8"):
+        pass
+    config_file = "minimal_config.ert"
+    with open(config_file, "w", encoding="utf-8") as f:
+        f.write("NUM_REALIZATIONS 1")
+        if design_matrix_entry:
+            f.write(
+                f"\nDESIGN_MATRIX {xls_filename} DESIGN_SHEET:DesignSheet01 DEFAULT_SHEET:DefaultValues"
+            )
+
+    args_mock = Mock()
+    args_mock.config = config_file
+
+    ert_config = ErtConfig.from_file(config_file)
+    with StorageService.init_service(
+        project=os.path.abspath(ert_config.ens_path),
+    ):
+        gui = _setup_main_window(ert_config, args_mock, GUILogHandler(), storage)
+        experiment_panel = gui.findChild(ExperimentPanel)
+        assert isinstance(experiment_panel, ExperimentPanel)
+
+        simulation_mode_combo = experiment_panel.findChild(QComboBox)
+        assert isinstance(simulation_mode_combo, QComboBox)
+
+        simulation_mode_combo.setCurrentText(EnsembleExperiment.name())
+        simulation_settings = gui.findChild(EnsembleExperimentPanel)
+        show_dm_parameters = simulation_settings.findChild(
+            QPushButton, "show-dm-parameters"
+        )
+
+        assert isinstance(show_dm_parameters, QPushButton)
+        assert show_dm_parameters.isEnabled() == design_matrix_entry
+        if show_dm_parameters.isEnabled():
+            design_matrix_df = pd.DataFrame({"a": [1, 2, 3], "b": [3, 4, 5]})
+            with patch.object(
+                DesignMatrix, "design_matrix_df", design_matrix_df
+            ), patch.object(DesignMatrix, "xls_filename", Path(xls_filename)):
+                qtbot.mouseClick(show_dm_parameters, Qt.LeftButton)
```

Comment:
Can we also assert the new component is showing?

Reply:
I have removed it